test/views/UploadListPageSpec.scala

```diff
@@ -29,22 +29,39 @@ import viewtils.RadioOptionHelper
 
 class UploadListPageSpec extends SpecBase with ViewBehaviours {
```

Comment:
Could we also have a test for the inset text not appearing when there is no duplicates? Maybe this could be added to an existing test rather than creating a new one?